app/src/main/java/org/mozilla/fenix/home/recenttabs/interactor/RecentTabInteractor.kt

```diff
@@ -17,6 +17,11 @@ interface RecentTabInteractor {
      */
     fun onRecentTabClicked(tabId: String)
 
+    /**
+     * Called when opening the dropdown menu on a recent tab by long press.
```

Comment:
```suggestion
     * Called when a user long clicks on a recent tab.
```
Removing the dropdown menu bit since we didn't add that above. Also, maybe the dropdown menu bit might assume too much implementation detail given it's just an interface.